registry/config.go

```diff
@@ -143,6 +143,13 @@ func (config *serviceConfig) LoadInsecureRegistries(registries []string) error {
 
 skip:
 	for _, r := range registries {
+		if strings.Contains(r, "://") {
```

Comment:
looks like we have a function for this; `validateNoScheme()`, should that be used?